routing/routing_tests/turns_sound_test.cpp

```diff
@@ -107,18 +107,17 @@ UNIT_TEST(TurnsSoundMetersTest)
   turnSound.Reset();
   turnSound.SetSpeedMetersPerSecond(30.);
 
-  TurnItem turnItem(5 /* idx */, TurnDirection::TurnRight);
+  vector<TurnItemDist> turns = {{{5 /* idx */, TurnDirection::TurnRight}, 1000.}};
   vector<string> turnNotifications;
 
-  ASSERT(turnNotifications.empty(), ());
-
   // Starting nearing the turnItem.
   // 1000 meters till the turn. No sound notifications is required.
-  turnSound.GenerateTurnSound(turnItem, 1000. /* distanceToTurnMeters */, turnNotifications);
+  turnSound.GenerateTurnSound(turns, turnNotifications);
```

Comment:
Are you about 
   // Starting nearing the turnItem.
   // 1000 meters till the turn. No sound notifications is required.

and the comets below. I'd prefer to leave them as is. The thing is they separate the test into parts and explain step by step what is going on.